sig-api-machinery/annual-report-2024.md

```diff
@@ -3,23 +3,34 @@
 ## Current initiatives and Project Health
 
 1. What work did the SIG do this year that should be highlighted?
-
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+- Officially added Stefan Schimanski (sttts@) as the third tech lead of the SIG.
+- Carefully reviewed on each release the list of open KEPs and which ones were going to make it in the release.
+- 2024 Contributor Awards:
+  - Marek Siarkowicz, @serathius
+  - Lukasz Szaszkiewicz, @polynomial
+- Some KEPs that deserve special mention in 2024 are
+  - Graduated to *Stable*
+    - [CEL for Admission Control](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3488-cel-admission-control#summary)
+    - [Aggregated Discovery](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3352-aggregated-discovery#summary)
+    - [Unknown Version Interoperability Proxy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/4020-unknown-version-interoperability-proxy#summary)
+    - [Transition from SPDY to Websockets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/4006-transition-spdy-to-websockets#summary)
+    - [Custom Resource Field Selectors](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/4358-custom-resource-field-selectors#summary)
+  - Graduated to *Beta*
+    - [Consistent Reads from Cache](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#summary)
+    - [Allow informers for getting a stream of data instead of chunking](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3157-watch-list#summary)
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
-
+- API Machinery area is very extense, here are some areas that could use extra help:
+  - Garbage Collection
+  - Resource Quota
+  - Server Side Apply
```

Comment:
nit: would consider SSA as the most pressing. Could go to the top of the list.

Reply:
yes, good point! changed it

Reply:
Would it be useful to expand on what help these areas need?  I'm trying to reason through why we would include Resource Quota and not controller infrastructure / clients.

Maybe expand Garbage Collection to something more general like Resource Lifecycle? We've got SVM and Namespace deletion ordering on our plate, and the Gateway API leads suggest that CRD lifecycle is a problem for their extensibility needs.

Reply:
Agree with you, I just put what came to my mind, but I was hoping to get more details or ideas from all of you.